libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/PushNotificationManager.swift

```diff
@@ -111,14 +111,19 @@ extension PushNotificationManager {
     }
     
     private func fetchNotificationTypesFromAPI(with client: RestClient) async throws -> [NotificationType] {
-        guard client.apiVersion.compare("v61.0").rawValue >= 0 else {
-            throw PushNotificationManagerError.notificationActionInvocationFailed("API Version must be at least v61.0")
+        guard client.apiVersion.compare("v64.0").rawValue >= 0 else {
+            throw PushNotificationManagerError.notificationActionInvocationFailed("API Version must be at least v64.0")
         }
         
-        let request = RestRequest(method: .GET, path: "connect/notifications/types", queryParams: nil)
+        let request = RestRequest(method: .GET, path: "\(client.apiVersion)/connect/notifications/types", queryParams: nil)
         let response = try await client.send(request: request)
-        let result = try response.asDecodable(type: NotificationTypesResponse.self)
-        return result.notificationTypes
+        do {
+            let result = try JSONDecoder().decode(NotificationTypesResponse.self, from: response.data)
```

Comment:
Was hitting some JSON decodign issue when I hit the live endpoint.  So felt appropiratre to catch any future decocede issues as well.